go.mod

```diff
@@ -3,76 +3,88 @@ module github.com/fluxcd/flagger
 go 1.17
 
 require (
-	cloud.google.com/go/monitoring v0.1.0
-	github.com/Masterminds/semver/v3 v3.0.3
-	github.com/aws/aws-sdk-go v1.37.32
+	cloud.google.com/go/monitoring v1.5.0
+	github.com/Masterminds/semver/v3 v3.1.1
+	github.com/aws/aws-sdk-go v1.44.24
```

Comment:
We should probably switch to v2: github.com/aws/aws-sdk-go-v2

Reply:
We don’t have an AWS account for testing, so I’m going to switch to v2 in this PR. Whoever does the update must test manually the CloudWatch integration.